app/routes.js

```diff
@@ -107,8 +107,26 @@ var BreadcrumbMaker = require('../lib/js/breadcrumb_maker.js');
       this.children = [];
     }
 
+    /*
+    Currently we're tagging organisation home pages and about pages
+    ("corporate information pages") to the taxonomy.
+
+    We're not sure if this is the right thing to do because these pages are
+    about the organisation themselves, not about the topic, even though
+    the organisation may be relevant to the topic. For now, just ignore these
+    tags, and don't mix these pages in with the rest of the content.
+    This is a workaround until we change how the content is tagged.
+    */
+    isGovernmenty(basePath) {
+      return basePath.includes("/organisations/");
+    }
+
     addContent(content) {
-      this.content.push(content);
+      if (this.isGovernmenty(content.basePath)) {
+        console.log("Excluding organisation page from taxonomy: %s", content.title);
```

Comment:
I'm not sure about logging things all the time.  We know we're doing this, and when we go to try and debug things having logs all over the place for things we know are happening are just going to make things more confusing.